scripts/wattlab_parity_oracle_dump.py

```
@@ -21,14 +23,44 @@
 
 def _sha256(path: Path) -> str:
     h = hashlib.sha256()
     with path.open("rb") as f:
         for chunk in iter(lambda: f.read(1 << 20), b""):
             h.update(chunk)
-    return h.hexdigest()
+        return h.hexdigest()
+
+
+def _docker_running(name: str) -> bool:
+    r = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)